### PR TITLE
Update composer configuration to support Symfony 3

### PR DESCRIPTION
**Pull request by mickaelandrieu**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,16 +27,16 @@
     "php": ">=5.4",
     "PrestaShop/smarty": "dev-master",
     "nikic/php-parser": "^2.1",
-    "twig/twig": "^1.34",
-    "symfony/twig-bridge": "^2.8"
+    "twig/twig": "^1.34|^2.0",
```
Comment (Soullivaneuh) [inline composer.json]: The recommended official notation is `^1.34 || ^2.0`

Comment (mickaelandrieu) [reply composer.json]: you're right, thanks

**Comment by mickaelandrieu** [comment]
> Symfony 4 should be out today. Maybe you can upgrade to this version too.

Really risky, I'm in favor of waiting for Symfony 4.1 :)

**Comment by Quetzacoalt91** [comment]
Sorry @mickaelandrieu, but tests don't work anymore with your PR.

**Comment by mickaelandrieu** [comment]
if tests pass, let me time to rebase and clean this history :)

**Comment by Quetzacoalt91** [comment]
Nupe, still failing :(

**Pull request merged**